packages/imperative/src/config/src/Config.ts

```diff
@@ -179,7 +179,9 @@ export class Config {
                 exists: false,
                 properties: Config.empty(),
                 global: layer === Layers.GlobalUser || layer === Layers.GlobalConfig,
-                user: layer === Layers.ProjectUser || layer === Layers.GlobalUser
+                user: layer === Layers.ProjectUser || layer === Layers.GlobalUser,
+                ignoreErrors: process.argv.includes("--help") || process.argv.includes("--version")
+                ||  process.argv[process.argv.length-1] === require.resolve('@zowe/cli')
```

Comment:
I don't think we should have `process.argv` checks hardcoded in the `Config.ts` file because it may be used by extenders like Zowe Explorer and we don't want to introduce unexpected side effects based on the CLI args passed to a 3rd party app that uses our SDKs.

Could the `ignoreErrors` flag be added as a property on the `IConfigOpts` interface, and passed in to the `Config.load` method from `Imperative.init`?

Reply:
okay i think i made this change @t1m0thyj , let me know if i messed up